src/client/client.ts

```diff
@@ -172,6 +175,27 @@ export class Client extends BaseClient {
     return this.fetchResources("passportScores", {}, passportScoreBuilder);
   }
 
+  getMatchingEstimates(
+    roundId: string,
+    chainId: string,
+    potentialVotes: {
+      contributor: string;
+      recipient: string;
+      amount: bigint;
```

Comment:
Does this account for https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/BigInt#use_within_json ?